tutorial/01_Introduction/intro.ipynb

```diff
@@ -8,7 +8,7 @@
    "source": [
```

Comment:
<p>do you plan to implement units here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hpc4cmb/toast/pull/369/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Good idea.  astropy.units are a new addition to toast, and currently only used in the new Operator traits.  I need to systematically go through the codebase and add support.</p>

Reply:
<p>I converted the fake focalplane simulation and plotting functions to use units.  However, I'll wait on the rest of the instrument classes until we can revisit the overall plan for those.</p>